typesafe/src/test/scala/zio/config/typesafe/TypesafeConfigErrorsSpec.scala

```diff
@@ -101,6 +101,17 @@ object TypesafeConfigErrorsSpec extends DefaultRunnableSpec {
       assert(nestedConfigAutomaticResult3)(equalTo(nestedConfigAutomaticExpect3)) &&
       assert(nestedConfigAutomaticResult4)(equalTo(nestedConfigAutomaticExpect4))
     },
+    test("A variant error case with a not well-formed typesafe HOCON config") {
+      val hocconStringWithParseError =
+        s"""
+         account {
+        """
+
+      val notWellFormedConfigResult = TypesafeConfigSource.fromHoconString(hocconStringWithParseError)
+      val notWellFormedConfigExpect = Left("String: 3: expecting a close parentheses ')' here, not: end of file")
+
+      assert(notWellFormedConfigResult)(equalTo(notWellFormedConfigExpect))
```

Comment:
This test is very specific. I would go for something like:
```scala
assert(notWellFormedConfigResult.isLeft)(Assertion.isTrue)
```

Reply:
Agreed. Modification done.